client/src/components/QA/QAContext.jsx

```diff
@@ -48,7 +48,13 @@ export function QADataProvider({ children }) {
   // const allQAData = useMemo(() => ({}))
   return (
     // eslint-disable-next-line react/jsx-no-constructed-context-values
-    <QAContext.Provider value={{ qData, aData }}>
+    <QAContext.Provider value={{
+      qData,
```

Comment:
This works, but might be more "react-y" to follow the custom hooks pattern and using a custom `useQData` and `useAData` hook

Reply:
Wait, I looked at the rest of your code and that's exactly what you're doing. Ignore this!